server/src/test/python/cromwell-az-e2e-test/cromwell_az_e2e_test/az_e2e.py

```diff
@@ -0,0 +1,173 @@
+#    workspaceId = create_workspace_data['workspaceId']
+
+#    print(f"Enabling CBAS for workspace {workspaceId}")
+#    activate_cbas_request = f"{leo_url}/api/apps/v2/{workspaceId}/terra-app-{str(uuid.uuid4())}"
+#    cbas_request_body = {
+#       "appType": "CROMWELL"
+#    } 
+        
+#    response = requests.post(url=activate_cbas_request, json=cbas_request_body, 
+#                             headers={"Authorization": f"Bearer {bearer_token}"})
+#    # will return 202 or error
+#    handle_failed_request(response, "Error activating CBAS", 202)
+   
+#    print(response)
+#    return workspaceId
+
+# # GET WDS OR CROMWELL ENDPOINT URL FROM LEO
+# def get_app_url(workspaceId, app):
+#     """"Get url for wds/cbas."""
+#     uri = f"{leo_url}/api/apps/v2/{workspaceId}?includeDeleted=false"
+
+#     headers = {"Authorization": bearer_token,
+#                "accept": "application/json"}
+
+#     response = requests.get(uri, headers=headers)
+#     status_code = response.status_code
+
+#     if status_code != 200:
+#         return response.text
+#     print("Successfully retrieved details.")
+#     response = response.json()
+
+#     app_url = ""
+#     app_type = "CROMWELL" if app != 'wds' else app.upper()
+#     print(f"App type: {app_type}")
+#     for entries in response: 
+#         if entries['appType'] == app_type and entries['proxyUrls'][app] is not None:
+#             print(entries['status'])
+#             if(entries['status'] == "PROVISIONING"):
+#                 print(f"{app} is still provisioning")
+#                 break
+#             print(f"App status: {entries['status']}")
+#             app_url = entries['proxyUrls'][app]
+#             break 
+
+#     if app_url is None: 
+#         print(f"{app} is missing in current workspace")
+#     else:
+#         print(f"{app} url: {app_url}")
+
+#     return app_url
+
+def submit_workflow_to_cromwell(app_url, workflow_test_name):
+    absolute_file_path = os.path.dirname(__file__)
+    workflow_source_path = os.path.join(absolute_file_path, 'workflow_files/hello.wdl')
+    workflow_inputs_path = os.path.join(absolute_file_path, 'workflow_files/hello.inputs')
+    workflow_endpoint = f'{app_url}/api/workflows/v1'
+    headers = {"Authorization": f'Bearer {bearer_token}',
+              "accept": "application/json",
+    }
+    files = {
+        'workflowSource': ('hello.wdl', open(workflow_source_path, 'rb'), 'application/octet-stream'),
+        'workflowInputs': ('hello.inputs', open(workflow_inputs_path, 'rb'), 'application/octet-stream'),
+        'workflowType': 'WDL',
+        'workflowTypeVersion': '1.0'
+    }
+    response = requests.post(workflow_endpoint, headers=headers, files=files)
+    handle_failed_request(response, f"Error submitting workflow to Cromwell for {workflow_test_name}", 201)
+    output_message(response.json())
+    return response.json()
+
+def get_workflow_information(app_url, workflow_id):
```

Comment:
If you're just looking for status information, you can get it from `/api/workflows/v1/{workflow_id}/status`.